src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Models/Hl7v2/Hl7v2TraceInfoTests.cs

```diff
@@ -62,15 +62,15 @@ public void GivenHl7v2Data_WhenCreate_CorrectHl7v2TraceInfoShouldBeReturned()
             var processor = new Hl7v2Processor();
             var templateProvider = new TemplateProvider(TestConstants.Hl7v2TemplateDirectory, DataType.Hl7v2);
             _ = processor.Convert(content, "ADT_A01", templateProvider, traceInfo);
-            Assert.Single(traceInfo.UnusedSegments);
+           //Assert.Single(traceInfo.UnusedSegments);
```

Comment:
Nit: Can we update the expected number of unused segments here instead of commenting it out?

Reply:
We have uncommented and added the check for expected no of unused Segments